ui/app/AppLayouts/Wallet/panels/SwapInputPanel.qml

```diff
@@ -49,8 +49,17 @@ Control {
 
     // FIXME drop after using ModelEntry, shouldn't be needed
```

Comment:
Do you know if we have a task for this? 🤔  Or maybe it's a good time now to do it?

Reply:
Ticket created: https://github.com/status-im/status-desktop/issues/16374